<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sCheckPass</class>
 <widget class="QWidget" name="sCheckPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frPassword">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>80</y>
     <width>650</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 3px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30px;

color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;

background-color: rgb(241, 241, 241);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;
font: 75 16pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="sQLEExtend" name="lePassword">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>300</width>
      <height>42</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 1px 18px 1px 10px;
background-color: rgb(255,255,255);
border-width: 2px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 15;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;

</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblCaption">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Roboto Medium&quot;;
background-color: rgb(21, 100, 192);
color: rgb(255, 255, 255);
border-radius : 0px;
border-top-left-radius: 30px;
border-top-right-radius: 30px;
border-width:0px;



</string>
    </property>
    <property name="text">
     <string>General Setup</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConfirm">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(120, 200, 90);
/*border-width: 2px;
border-color: rgb(120, 200, 90);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(190, 190, 190);;
/*border-width: 2px;
border-color: rgb(190, 190, 190);;
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(255, 255, 255);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
/*border-width: 2px;
border-color: rgb(241, 241, 241);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCan">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(240,90,40);
/*border-width: 2px;
border-color: rgb(240,90,40);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0,0,0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(190, 190, 190);
/*border-width: 2px;
border-color: rgb(190, 190, 190);
border-style: solid;*/
    border: none;
    outline: none;
border-radius: 20;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(194, 54, 11);
/*border-width: 2px;
border-color: rgb(194, 54, 11);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(255, 255, 255);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
/*border-width: 2px;
border-color: rgb(241, 241, 241);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;
font: 75 16pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="sQLEExtend" name="leUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>300</width>
      <height>42</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 1px 18px 1px 10px;
background-color: rgb(255,255,255);
border-width: 2px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 15;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;

</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>660</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 3px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30px;

color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;

background-color: rgb(241, 241, 241);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblMessage">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>83</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;
font: 75 16pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>Wrong Password!
Enter valid password &amp; try again!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblMCaption">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>660</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Roboto Medium&quot;;
background-color: rgb(21, 100, 192);
color: rgb(255, 255, 255);
border-radius : 0px;
border-top-left-radius: 30px;
border-top-right-radius: 30px;
border-width:0px;


</string>
    </property>
    <property name="text">
     <string>General Setup</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOK">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>165</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(120, 200, 90);
/*border-width: 2px;
border-color: rgb(120, 200, 90);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(190, 190, 190);;
/*border-width: 2px;
border-color: rgb(190, 190, 190);;
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(255, 255, 255);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
/*border-width: 2px;
border-color: rgb(241, 241, 241);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frConfirm">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>670</y>
     <width>660</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 3px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30px;

color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;

background-color: rgb(241, 241, 241);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblCMessage">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>63</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;
font: 75 16pt &quot;Roboto Medium&quot;;</string>
    </property>
    <property name="text">
     <string>Do you want to delete?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCCaption">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>660</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Roboto Medium&quot;;
background-color: rgb(21, 100, 192);
color: rgb(255, 255, 255);
border-radius : 0px;
border-top-left-radius: 30px;
border-top-right-radius: 30px;
border-width:0px;


</string>
    </property>
    <property name="text">
     <string>Memory Delete</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbYes">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>125</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(120, 200, 90);
/*border-width: 2px;
border-color: rgb(120, 200, 90);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(190, 190, 190);;
/*border-width: 2px;
border-color: rgb(190, 190, 190);;
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(255, 255, 255);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
/*border-width: 2px;
border-color: rgb(241, 241, 241);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>125</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(240,90,40);
/*border-width: 2px;
border-color: rgb(240,90,40);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0,0,0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(190, 190, 190);
/*border-width: 2px;
border-color: rgb(190, 190, 190);
border-style: solid;*/
    border: none;
    outline: none;
border-radius: 20;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(194, 54, 11);
/*border-width: 2px;
border-color: rgb(194, 54, 11);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(255, 255, 255);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
/*border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
/*border-width: 2px;
border-color: rgb(241, 241, 241);
border-style: solid;*/
border-radius: 20;
    border: none;
    outline: none;
color: rgb(0, 0, 0);
font: 100 18pt &quot;Roboto Medium&quot;;
}</string>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="imageCapture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>795</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    outline: none; /* Ensures no outline is displayed */
}
QPushButton:pressed {
    border: none;
    outline: none; /* Ensures no outline is displayed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sQLEExtend</class>
   <extends>QLineEdit</extends>
   <header location="global">sqleextend.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
